### ISSUE TITLE
segfault in barcode_collapse_pe.py when output dir doesn't exist

### ISSUE THREAD

**Issue by JCSzamosi** [YeoLab/gscripts#92]

**Comment by gpratt** [comment]
Thanks for letting me know! 

I'm planning on moving away from this code soon and switching to UMI tools so I probably won't fix it.  Let me know if it is blocking for you though.

**Comment by JCSzamosi** [comment]
Nope, it's not in my way. Once I figured out what was happening, mkdir fixed it.
